bin/rename-files

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env ruby
+require 'fileutils'
+
+class String
+  def param_case
+    gsub(/::/, "/")
+      .gsub(/([A-Z]+)([A-Z][a-z])/, '\1_\2')
+      .gsub(/([a-z\d])([A-Z])/, '\1_\2')
```

Comment:
I might be wrong, but it looks to me like this will do snake case rather than param case.

I would have expected the replacement to be `'\1-\2'` rather than '\1_\2'.